config/routes.rb

```diff
@@ -194,7 +194,11 @@
       get :podcasts
     end
 
-    resource :onboarding, only: :show
+    scope module: "users" do
+      resource :onboarding, only: %i[show update]
+      patch "/onboarding_checkbox_update", to: "onboardings#onboarding_checkbox_update"
```

Comment:
I would like to combine `#update` and `#onboarding_checkbox_update` but I might be making thing's trickier than it needs to be.

Reply:
Just confirming this _didn't_ happen in this PR (the checkbox update route still exists, and is still used in some slides)